frontend/src/pages/DoctorEditProfilePage.jsx

```
@@ -1,141 +1,416 @@
+//             }
+//         } catch (error) {
+//             toast.error('Failed to deactivate account');
+//         } finally {
+//             setIsDeactivating(false);
+//             setShowDeactivateModal(false);
+//         }
+//     };
+
+
+
+import { useState, useEffect } from "react";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'useState'. Consider to delete it or rename it.

'useState' is defined here:


(lint/suspicious/noRedeclare)